docs/reference/versions/index.md

```diff
@@ -1,35 +1,26 @@
 # Service version matrix
 
-Services in {{brand}} are updated on a regular basis and
-on a rolling schedule.
+Services in {{brand}} are updated on a regular basis and on a rolling schedule.
 
-This section lists the cloud API service versions available in each
-{{brand}} region.
+This section lists the cloud API service versions available in each {{brand}} region.
 
-* [Public Cloud](public.md): versions running in our {{company}}
-  Public Cloud regions.
+* [Public Cloud](public.md): versions running in our {{company}} Public Cloud regions.
 
-* [Compliant Cloud](compliant.md): versions running in our
-  {{company}} Compliant Cloud regions.
+* [Compliant Cloud](compliant.md): versions running in our {{company}} Compliant Cloud regions.
 
 
 ## OpenStack Services
 
-[OpenStack releases](https://releases.openstack.org) are named, in
-alphabetical order, and occur on a six-month release schedule. In
-{{company}} Public Cloud we upgrade OpenStack releases annually; this
-means that we deploy every other OpenStack release and skip the
-intervening one.
+[OpenStack releases](https://releases.openstack.org) are named in alphabetical order, and occur on a six-month release schedule.
+In {{company}} Public Cloud we upgrade OpenStack releases annually; this means that we normally deploy every other OpenStack release and skip the intervening one.
```

Comment:
We can tell even that we're tracking SLURP releases, to make an expectation what next release will be.

Reply:
I'd say it does help. First, we can add a link to the governance decision explaining how SLURP is different. This resolution also describes periodity of SLURP releases. Then, we're mentioning, that we're doing only officially supported upgrades by the community and aiming to follow best practices rather then weirdly jumping through releases.

Also, I don't see much sense tracking down upstream docs, when we all know that next SLURP release will be 2024.1(Caraval) and it will marked as one as soon as it will become in Development (or Maintained - not sure when they add SLURP tag to the page tbh).